sailfishusermanager.cpp

```diff
@@ -211,6 +211,14 @@ uint SailfishUserManager::addUser(const QString &name)
         return 0;
     }
 
+    if (users().count() > SAILFISH_USERMANAGER_MAX_USERS) {
```

Comment:
This does needlessly make calls to getpwnam so it would probably be better to call getgrnam directly and check how many users there are there. (Although, those getpwnam calls would be more useful if they also checked that the user actually exists but still it might have some effect to time complexity.)

Reply:
Fixed, also found the bug in users() method.